### ISSUE TITLE
Evaluation results on DTU dataset

### ISSUE THREAD

**Issue by ERGOWHO** [Mercerai/PFGS#6]
Hi, I cloned the latest version of the code and followed the instructions in the README. I downloaded the DTU dataset from the provided link and trained both stage 1 and stage 2 with the original PFGS code. However, the evaluation results are much lower than those reported in the paper.

Here are the evaluation results I obtained:

PSNR: 19.1
SSIM: 0.6461
LPIPS: 0.1618
Then, I downloaded the pretrained model for DTU provided in the README and ran the evaluation directly. The results were:

PSNR: 21.48
SSIM: 0.7536
LPIPS: 0.1145
These are still much lower than the values reported in the paper:

PSNR: 25.44
SSIM: 0.901
LPIPS: 0.164

I would like to know if I made any mistakes in the process. Could you please help me identify if I made any mistakes or missed any critical steps?
Thank you in advance for your help!

Below are the commands I used in my train.sh:
//Train Stage 1
python train_stage1.py --dataset dtu --scene_dir /data1/PFGS/dtu --exp_name abl_lmim_dtu_stage1 --img_wh 640 512 --resume_path logs/abl_lmim_dtu_stage1/lightning_logs/version_0/checkpoints/last.ckpt

//Train Stage 2
python train_stage2.py --dataset dtu --scene_dir /data1/PFGS/dtu --exp_name abl_lmim_dtu_stage2 --img_wh 640 512 --ckpt_stage1 logs/abl_lmim_dtu_stage1/lightning_logs/version_0/checkpoints/last.ckpt

//Evaluate Stage 2
python train_stage2.py --dataset dtu --scene_dir /data1/PFGS/dtu --exp_name abl_lmim_dtu_stage2_eval --img_wh 640 512 --resume_path logs/abl_lmim_dtu_stage2/lightning_logs/version_0/checkpoints/last.ckpt --val_mode test

// Evaluate Pretrained Model
python train_stage2.py --dataset dtu --scene_dir /data1/PFGS/dtu --exp_name pretrained_eval --img_wh 640 512 --resume_path /data1/PFGS/checkpoint/dtu_best.ckpt --val_mode test

**Comment by ERGOWHO** [comment]
I repeat did eval on official pretrained model for several times using `python train_stage2.py --dataset dtu --scene_dir /data1/yhu57782/PFGS/dtu --exp_name pretrain_eval --img_wh 640 512 --resume_path /data1/yhu57782/PFGS/checkpoint/dtu_best.ckpt --val_mode test,` without changing anything, the eval results seems to be very indifferent each time:
first time:
PSNR: 21.48
SSIM: 0.7536
LPIPS: 0.1145
second time:

![Image](https://github.com/user-attachments/assets/73b8206d-d147-4fc8-810b-e12a8194edca)
3rd time:

![Image](https://github.com/user-attachments/assets/9c0a95f0-ecb7-465b-8bf0-b2f7f10fc1dc)

and repeat 4th time:

![Image](https://github.com/user-attachments/assets/75696fe6-4dfb-47c8-9df9-1024ec644569)

still can not reach 25.44, is there anyone can give any suggestion on how should I eval to get similar number to paper:

![Image](https://github.com/user-attachments/assets/bc40287f-3d42-4e80-90d7-059c2e14c75a)

**Comment by ERGOWHO** [comment]
same problem when test on scannet pretrained model and scannet dataset.

**Comment by Mercerai** [comment]
Hello, Thanks for your attention. The DTU dataset we provide is only a subset of our processed test set, intended to demonstrate the format of our processed data. You need to process the remaining DTU data in the same format, then train on the training set and test on the test set.

For the SCANNET dataset, to enable quick visualization, we randomly selected a few scenes for evaluation in the public code. However, you should compute the metrics on the entire test set rather than just the scenes used for visualization.

**Comment by ERGOWHO** [comment]
Thank you for your reply, could you please provide which subset of DTU you used for training and testing?

**Comment by Mercerai** [comment]
We follow the MVSNeRF's train-test split. The MVSNeRF config file shows which scenes are used to train and test the model.
